### PR TITLE
Add Spanish internationalization

### PR DESCRIPTION
**Pull request by MrBenJ**
Hey, I didn't see a PR from @CRomano31415 or @fportela-ns so I figured I'd go ahead and make the PR.

However, I do NOT deserve all the credit. @CRomano31415 scraped all these words together and @fportela-ns combed through and assigned the appropriate numerical values for all of this. I just turned it into JSON and added maybe 2 or 3 words. My contribution is minimal compared to the work of these two individuals.

Yay internationalization!

### CODE REVIEW

File: es.json [hunk 1]
```diff
@@ -0,0 +1,559 @@
...
+  "zángano": 1,
+  "zarrapastroso": 2,
+  "zascandil": 2,
+  "zopenco": 2,
+  "zoquete": 2,
+  "zorra": 2,
+  "zote": 2,
+  "zullenco": 2,
+  "zurcefrenillos": 2
+}
```
Comment (wooorm) [inline es.json]: Newline here!

**Comment by MrBenJ** [comment]
No worries @wooorm - Feel free to close this and give the credit to the others ;). I already got my 5 PR's so the others should definitely get theirs ;).

**Comment by wooorm** [comment]
In that case, @CRomano31415 or @fportela-ns, need a PR?

**Comment by fportela-ns** [comment]
I'm good, this is my work account anyways and it's not registered for the hacktoberfest 😄

**Comment by CRomano31415** [comment]
I honestly would be okay if either @MrBenJ or @fportela-ns got it - no worries :)

**Comment by MrBenJ** [comment]
You're all wonderful and polite people :D. Open Source, full of nice people!

File: es.json (hunk 1 above)
Comment (MrBenJ) [reply es.json]: Newline added!

**Comment by wooorm** [comment]
@CRomano31415 @fportela-ns Awesome, thanks for getting back to us!

@MrBenJ Could you add Spanish to the [Language support section]https://github.com/words/cuss#language-support), and add something to [Support](https://github.com/words/cuss#support) about where this list came from?

**Comment by MrBenJ** [comment]
Donion rings!

**Pull request merged**